sonoff/i18n.h

```diff
@@ -193,6 +193,8 @@
 
 #define D_LOG_SOME_SETTINGS_RESET "Some settings have been reset"
 
+#define D_SENSOR_DEEPSLEEP "DeepSleep Switch"
+
```

Comment:
Text is too long. It must fit in the sensor drop down list which is also language specific. 

So D_SENSOR_DEEPSLEEP needs to move to every langugage file and have a default name like "DeepSleep"

Reply:
Stefan, I'll make some changes and will merge your code if I've done it.

There are some RTCMem issues too.

No need for you to do anything now.

Reply:
"DeepSleep" for the GPIO is fine. Let's wait how this gets translated into other languages by the community. RTCMem needs recheck for the "free" bytes. You will do it right. Besides this I got no problems here with the "pre"-core.

Reply:
Maybe you just comment out all the MORE_DEBUG messages to save space. No need for it.